md-render/HHVM/UserDocumentation/IncludeExamplesFilter.rb

```diff
@@ -51,7 +51,21 @@ def call
             File.read(full_path),
             language: 'Hack',
           );
-          node.replace(code)
+          code_node = node.replace(code)
+          # If the example has HHVM output associated with it, then print it
+          # after the code example
+          if File.exists? (full_path + ".example.hhvm.out")
```

Comment:
Yeah, avoiding the copy would be nice. It would also be good if we could check that the .example.hhvm.out match the expectf/expectre